projects/ui/src/shim.cds-core.scss

```diff
@@ -1,68 +0,0 @@
-/*
- * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
- * This software is released under MIT license.
- * The full license information can be found in LICENSE in the root directory of this project.
- */
-@use '@cds/core/tokens/tokens.scss';
-/* This shim file maps the custom properties of @clr/ui to Core tokens. Thus, it is only needed for apps currently dependent on @clr/ui and wanting to apply Core look and feel. */
-
-// Line height tokens are implemented as the smallest version in the figma specs but some
-// styles have a standard and/or expanded styles with larger line heights.
-@function convertCompactToStandardLineHeight($variable, $fallback_variable) {
-  @return calc(#{$variable} + (calc(4 / 20 * 1rem)));
-}
-
-// Reevaluate pixel values based on baseline rem setting.
-@function baselinePx($pixels: 0) {
-  @return calc($pixels * (1rem / var(--clr-global-font-size)));
-}
-
-[cds-theme] {
-  /* Global */
-  --clr-global-font-size: #{tokens.$cds-global-base};
```

Comment:
It seems we're dropping this.
I guess it's safe to remove, as it was used in typography props only in v15, but we may need a Breaking Change note in the pr commit for it.

Reply:
Updated the breaking changes notes